trlx/trainer/accelerate_base_trainer.py

```diff
@@ -468,18 +475,40 @@ def learn(self):  # noqa: C901
         for _ in range(self.config.train.epochs):
             # For each batch
             for batch in self.train_dataloader:
+                mbs = [
+                    PPORLBatch(
```

Comment:
@Dahoas Just to confirm, this is still a draft correct? Since (although I haven't run it) this I think would break for ILQLBatch and other types of datatypes

Reply:
Ah you're right, I just ran this with the benchmarking scripts and it crashed here with this error

Reply:
Fixed this here: https://github.com/CarperAI/trlx/pull/403